### PR TITLE
[ACS-5343] refactor Playwright framework in ACA

### PR DESCRIPTION
**Pull request by MichalFidor**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**
Now we are able to execute the tests with the nx command. The code is placed in the right places. Things has been cleaned up.

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by MichalKinas — APPROVED** [review]

File: e2e/playwright/folder-rules/playwright.config.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { PlaywrightTestConfig } from '@playwright/test';
+import { CustomConfig, getGlobalConfig, getExcludedTestsRegExpArray } from '@alfresco/playwright-shared';
+import EXCLUDED_JSON from './exclude.tests.json';
```
[the remark this answers is not in the document]
Comment (MichalFidor) [reply e2e/playwright/folder-rules/playwright.config.ts:27]: It's a default import, not a named one. `EXCLUDED_JSON` need to be imported without curly braces.

File: e2e/playwright/actions/src/tests/create-folder.spec.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ */
+
+import { expect } from '@playwright/test';
+import { getUserState, test } from '@alfresco/playwright-shared';
+
+test.use({ storageState: getUserState('hruser') });
+test.describe('Create actions', () => {
+  const randomFolderName = `playwright-folder-${(Math.random() + 1).toString(36).substring(6)}`;
+
+  test.beforeEach(async ({ personalFiles }) => {
```
[the remark this answers is not in the document]
Comment (azakrzewski-hy) [reply e2e/playwright/actions/src/tests/create-folder.spec.ts:32]: After talk with @MichalKinas delete manual action was added into afterEach function. Commit: https://github.com/Alfresco/alfresco-content-app/pull/3261/commits/98a2319739d0ef4d6d5b71d7a678ebf613b21b16

File: e2e/playwright/actions/playwright.config.ts [hunk 3]
```diff
@@ -22,17 +22,23 @@
  * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { Page } from '@playwright/test';
-import { BaseComponent } from '../base.component';
+import { PlaywrightTestConfig } from '@playwright/test';
+import { CustomConfig, getGlobalConfig, getExcludedTestsRegExpArray } from '@alfresco/playwright-shared';
+import EXCLUDED_JSON from './exclude.tests.json';
```
[the remark this answers is not in the document]
Comment (MichalFidor) [reply e2e/playwright/actions/playwright.config.ts:27]: https://github.com/Alfresco/alfresco-content-app/pull/3261#discussion_r1226488507

**Review by MichalFidor — COMMENTED** [review]
@AkashBurman you need to change the way how you are running the tests on GHActions here : https://github.com/Alfresco/alfresco-content-app/pull/3261/files#diff-fc79da000cf778fb8dda1c2f9847c2c4ae107459e10b85a04ee7eac531cc4082R51

It's no longer `npx playwright test --config <path>`, it won't work. As you can see there are still errors with lack of users data. You need to change it, to execute the Playwright tests by using `nx run <project>:e2e` command. This is the only way how you can do that. I advise you, to create a new action yml file, which will be only for Playwright and there do all the logic.

**Pull request merged**